src/JsonSchema/Fuzz.elm

```diff
@@ -99,30 +101,54 @@ propertyFuzzer objectProperty =
 
 arrayFuzzer : ArraySchema -> Fuzzer Value
 arrayFuzzer arraySchema =
-    case ( arraySchema.items, arraySchema.minItems, arraySchema.maxItems ) of
-        ( Nothing, _, _ ) ->
+    case ( arraySchema.items, arraySchema.minItems, arraySchema.maxItems, arraySchema.uniqueItems ) of
+        ( Nothing, _, _, _ ) ->
             -- TODO: do something more interesting in the case the user hasn't defined a schema for the items.
             Fuzz.constant []
                 |> Fuzz.map Encode.list
 
-        ( Just subSchema, Nothing, Nothing ) ->
+        ( Just subSchema, Nothing, Nothing, False ) ->
             schemaValue subSchema
                 |> Fuzz.list
                 |> Fuzz.map Encode.list
 
-        ( Just subSchema, Just minItems, Nothing ) ->
+        ( Just subSchema, Nothing, Nothing, True ) ->
+            schemaValue subSchema
+                |> Fuzz.list
+                |> Fuzz.map (\lv -> List.map (Json.Encode.encode 0) lv |> Set.fromList |> Set.toList |> List.map Json.Encode.string)
```

Comment:
❔ I don't complete understand what's happening here. Are we encoding every item twice? And how are we sure two json objects with equal keys in different orders are recognised as being the same object?